test/Cargo.toml

```diff
@@ -13,6 +13,9 @@ caliptra_common = { workspace = true, default-features = false }
 caliptra-hw-model-types.workspace = true
 caliptra-runtime = { workspace = true, default-features = false }
 openssl.workspace = true
+p384.workspace = true
```

Comment:
The emulator uses RustCrypto for most of the crypto, so to add diversity, I've been trying to use openssl for all the test vector generation in caliptra-test.